pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -202,6 +202,10 @@ protected Object internalGetObject(int columnIndex, Field field) throws SQLExcep
         return getTime(columnIndex);
       case Types.TIMESTAMP:
         return getTimestamp(columnIndex, null);
+//#if mvn.project.property.postgresql.jdbc.spec >= "JDBC4.2
+      case Types.TIMESTAMP_WITH_TIMEZONE:
+        return  getTimestamp(columnIndex, null);
```

Comment:
this probably needs to use the current timezone.

Also we should probably now deal with time with timezone